src/view/components/organisms/NftContainer.tsx

```diff
@@ -23,15 +23,36 @@ const NftContainer = (): JSX.Element => {
         const loadNFTs = async () => {
             if (wallet.walletAddress) {
                 //do not try to fetch if undefined
-                const nftsForOwner = await alchemy.nft.getNftsForOwner(wallet.walletAddress)
+                const nftsForOwner = await alchemy.nft.getNftsForOwner(
+                    wallet.walletAddress
+                )
                 console.log("number of NFTs found:", nftsForOwner.totalCount)
```

Comment:
Better to use a logging library like loglevel. You can set the loglevel in app.js and suppress dev logs in production